tests/test_alias.py

```diff
@@ -14,7 +15,8 @@ def on_get(self, req, resp):
 
 @pytest.fixture
 def alias_client():
-    api = falcon.API()
+    with pytest.warns(DeprecatedWarning, match='API class may be removed'):
```

Comment:
These are neat, I'm not sure why it wasn't done this way in the first place.

Reply:
not sure, I was chasing a warning, so I fixed a bunch along the way